__tests__/components/CTSIText.test.tsx

```diff
@@ -9,26 +9,29 @@
 // WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
 // PARTICULAR PURPOSE. See the GNU General Public License for more details.
 
-import { Text } from '@chakra-ui/react';
+import { Icon, Text } from '@chakra-ui/react';
 import { render, screen } from '@testing-library/react';
 import { FaIcons } from 'react-icons/fa';
 import CTSIText from '../../src/components/CTSIText';
+import { withChakraTheme } from '../test-utilities';
+
+const Component = withChakraTheme(CTSIText);
 
 describe('CTSI Text component', () => {
     it('Should render the formatted amount of CTSI', () => {
-        render(<CTSIText value="5000100000000000000000000" />);
+        render(<Component value="5000100000000000000000000" />);
```

Comment:
I had to refactor some of the tested component so that they are always augmented with the chakra-ui context. Otherwise those tests were failing.